api/account/api.go

```diff
@@ -77,17 +77,20 @@ type CredentialRepresentation struct {
 
 // Configuration struct
 type Configuration struct {
-	EditingEnabled                    *bool           `json:"editing_enabled"`
-	ShowAuthenticatorsTab             *bool           `json:"show_authenticators_tab"`
-	ShowPasswordTab                   *bool           `json:"show_password_tab"`
-	ShowProfileTab                    *bool           `json:"show_profile_tab"`
-	ShowAccountDeletionButton         *bool           `json:"show_account_deletion_button"`
-	RedirectSuccessfulRegistrationURL *string         `json:"redirect_successful_registration_url"`
-	AvailableChecks                   map[string]bool `json:"available-checks"`
-	BarcodeType                       *string         `json:"barcode_type"`
-	Theme                             *string         `json:"theme"`
-	SupportedLocales                  *[]string       `json:"supportedLocales,omitempty"`
-	ShowGlnEditing                    *bool           `json:"show_gln_editing,omitempty"`
+	EditingEnabled                        *bool           `json:"editing_enabled"`
+	ShowAuthenticatorsTab                 *bool           `json:"show_authenticators_tab"`
+	ShowPasswordTab                       *bool           `json:"show_password_tab"`
+	ShowProfileTab                        *bool           `json:"show_profile_tab"`
+	ShowAccountDeletionButton             *bool           `json:"show_account_deletion_button"`
+	RedirectSuccessfulRegistrationURL     *string         `json:"redirect_successful_registration_url"`
+	AvailableChecks                       map[string]bool `json:"available-checks"`
+	BarcodeType                           *string         `json:"barcode_type"`
+	Theme                                 *string         `json:"theme"`
+	SupportedLocales                      *[]string       `json:"supportedLocales,omitempty"`
+	ShowGlnEditing                        *bool           `json:"show_gln_editing,omitempty"`
+	VoucherEnabled                        *bool           `json:"voucher_enabled"`
```

Comment:
Do we have to expose in account API those 3 parameters ? voucher-enabled is not enough ?